packages/utils/src/test_utils/middleware/common.rs

```diff
@@ -221,3 +222,86 @@ impl AvsOperator {
         }
     }
 }
+
+pub async fn validate_docker_container_id(container_id: &str) -> Result<()> {
+    // Validate that container_id is a valid Docker container ID
+    // Docker container IDs are hexadecimal strings, typically 12 or 64 characters
+    if container_id.len() < 12 || container_id.len() > 64 {
+        bail!(
+            "Invalid container ID length: {} (expected 12-64 characters)",
+            container_id.len()
+        );
+    }
+
+    if !container_id.chars().all(|c| c.is_ascii_hexdigit()) {
```

Comment:
It should be lowercase as well https://stackoverflow.com/a/36863724

Reply:
The linked article was written ~9 years ago and suggests that it could change in the future.

The current regex to check validity allows for uppercase: https://github.com/opencontainers/go-digest/blob/89707e38ad1aab6815bde4ad095806212ec90236/digest.go#L66

Reply:
ah but then it does deeper validation ultimately here, which does require lowercase: https://github.com/opencontainers/go-digest/blob/89707e38ad1aab6815bde4ad095806212ec90236/algorithm.go#L163